docs/02_getting_started.md

```diff
@@ -37,10 +37,10 @@ You may even find some new, indispensable tools to help with continuous integrat
 
 **Git is:**
 
-- a distributed version control system (DVCS).
-- free and open source.
-- designed to handle everything from small to very large projects with speed and efficiency.
-- easy to learn and has a tiny footprint with lightning fast performance.
+- A distributed version control system (DVCS).
+- Free and open source.
+- Designed to handle everything from small to very large projects with speed and efficiency.
+- Small in size and fast.
```

Comment:
```suggestion
- A distributed version control system (DVCS)
- Free and open source
- Designed to handle everything from small to very large projects with speed and efficiency
- Small in size and fast
```

Reply:
@parkerbxyz so @stebje and I made a note of the `.` in the bullets here, but they're also everywhere in this page, and possibly all of the manual because it was written before those rules were more widely known and practiced. We can fix them here alone, fix them everywhere on the page, fix them everywhere in the manual, or leave them all as is for now. If nothing else, it's a good first pull request for the manual, which we could open a ticket for and allow someone the opportunity?

Reply:
I’m leaning toward fixing this list since we’re making other changes to it, but I’m OK with any of the options.

Reply:
Voting to fix it here, so will commit the suggestion. We can still open a PR to fix the rest, maybe do a sweep across the other pages as well in the manual 🙂